packages/beacon-node/test/spec/utils/specTestIterator.ts

```diff
@@ -59,6 +62,11 @@ export function specTestIterator(configDirpath: string, testRunners: Record<stri
       }
 
       for (const testHandler of readdirSyncSpec(testRunnerDirpath)) {
+        if (
+          ["full_withdrawals", "partial_withdrawals", "bls_to_execution_change", "withdrawals"].includes(testHandler)
```

Comment:
This will be removed once new spec tests land right?

Reply:
ummm, new test runners for them need to be written, is part of the TODO tracker here https://github.com/ChainSafe/lodestar/issues/4680